dotnet/src/webdriver/NoSuchElementException.cs

```diff
@@ -69,7 +70,7 @@ public NoSuchElementException(string message, Exception innerException)
         /// </summary>
         /// <param name="message">The original message for exception</param>
         /// <returns>The final message for exception</returns>
-        protected static string GetMessage(string message)
+        protected static string GetMessage(string? message)
         {
             return message + "; " + supportMsg + supportUrl;
```

Comment:
`null` + "abc" - not sure it works.

Reply:
I converted this method to interpolated string as well.